### PR TITLE
Implementing Acts terms inclusion and alignments

### LINKED ISSUE #396: Some terms need to be aligned with EUDIW Implementing Acts
‘wallet secure cryptographic application’ means an application that manages critical assets by using the cryptographic functions provided by the wallet secure cryptographic device; 

‘wallet instance’ means the application installed and configured on a wallet user’s device or environment, which is part of a wallet unit, and that the wallet user uses to interact with the wallet unit; 

‘wallet unit’ means a unique configuration of a wallet solution that includes wallet instances, wallet secure cryptographic applications and wallet secure cryptographic devices provided by a wallet provider to an individual wallet user; 

‘wallet unit attestation’ means a data object that describes the components of the wallet unit, allow authentication and validation of those components and are cryptographically bound to wallet secure cryptographic devices;

### PR DESCRIPTION
**Pull request by peppelinux**
this PR introduces new peculiar terms taken from the eidas implementing acts.

These terms are useful for producing further details and improvements within the wallet solution architecture, that are kept out of the scope of this PR.

This PR resolves https://github.com/italia/eudi-wallet-it-docs/issues/396

### CODE REVIEW

**Comment by m-basili** [comment]
also related to #230

File: docs/en/defined-terms.rst [hunk 1]
```diff
@@ -115,8 +115,17 @@ Below are the description of acronyms and definitions which are useful for furth
...
-   * - Wallet Attestation
-     - Verifiable Attestation, issued by the Wallet Provider, that proves the security compliace of the Wallet Instance.
+   * - Wallet Secure Cryptographic Application
+     - An application that manages critical assets utilizing the cryptographic functions provided by the Wallet Secure Cryptographic Device.
+     -
+   * - Wallet Instance
+     - The application installed and configured on a Wallet User’s device or environment, which is part of a Wallet Unit, and that the Wallet User uses to interact with the Wallet Unit.
+     -
+   * - Wallet Unit
+     - Unique configuration of a wallet solution that includes wallet instances, wallet secure cryptographic applications, and wallet secure cryptographic devices provided by a wallet provider to an individual wallet user.
```
Comment (peppelinux) [inline docs/en/defined-terms.rst]:
```suggestion
     - Unique configuration of a Wallet Solution that includes Wallet instances, Wallet Secure Cryptographic Applications, and Wallet Secure Cryptographic Devices provided by a Wallet Provider to an individual Wallet User.
```

File: docs/en/defined-terms.rst [hunk 2]
```diff
@@ -115,8 +115,17 @@ Below are the description of acronyms and definitions which are useful for furth
...
+     -
+   * - Wallet Unit Attestation
+     - Also known as Wallet Attestation or Wallet Instance Attestation, it is a Data object issued by a Wallet Provider that describes the components of the Wallet Unit. It allows authentication and validation of those components, and is cryptographically bound to Wallet Secure Cryptographic Devices.
```
Comment (peppelinux) [inline docs/en/defined-terms.rst]:
```suggestion
     - Also known as Wallet Attestation or Wallet Instance Attestation, it is a data object issued by a Wallet Provider that describes the components of the Wallet Unit. It allows authentication and validation of those components, and is cryptographically bound to Wallet Secure Cryptographic Devices.
```

**Review by fmarino-ipzs — APPROVED** [review]

**Review by grausof — APPROVED** [review]
Ok for now, we will update this content regarding the issue https://github.com/italia/eudi-wallet-it-docs/issues/230 asap

**Pull request merged**